kamailio/local.cfg

```diff
@@ -106,6 +106,12 @@ mhomed=0
 ##!define KZ_UDP_REGISTRAR_PORT 7000
 ##!define KZ_TCP_REGISTRAR_PORT 7000
 
+################################################################################
+## NAT
+################################################################################
+## These parameters are OPTIONAL.
+# # #!substdef "!NAT_UAC_TEST_LEVEL!3!g"
```

Comment:
sorry for being picky. you can set here the value 1 (it is commented) as an example of overriding the default value. you can also describe that the default value is 3. somethink like
```
this parameter is OPTIONAL
it allows overriding the nat_uac_test with a different value than "3" (default) as it is proven that it fixes the issue reported here (add the url)
```